internal/pkg/service/common/configmap/flags.go

```diff
@@ -74,7 +74,7 @@ func StructToFlags(fs *pflag.FlagSet, v any, outFlagToField map[string]orderedma
 			case bool:
 				fs.Bool(flagName, v, usage)
 			case string:
-				if vc.Value.IsZero() {
+				if !vc.Value.IsValid() || vc.Value.IsZero() {
 					// Don't set the default Value, if the original Value is empty.
```

Comment:
Here, if a parent pointer was `nil`, the child value (that doesn't exists) is marked as invalid by the reflection.